### DISCUSSION TITLE
Representing wet lab provenance

### DISCUSSION THREAD

**Discussion by multimeric** [ResearchObject/ro-crate#351] [Q&A]
I've had a look at the [Provenance section of the spec](https://www.researchobject.org/ro-crate/specification/1.1/provenance.html), and I can generally see how to represent the final process that converted a physical specimen into a data file, using `CreateAction`. However, I also want to include all of the lab steps such as the sample preparation that preceded the sequencing, microscopy etc. Are there:

* Any examples of this that I can take inspiration from
* Any integrations with other specs such as from `BioSchemas` that can be used to represent this metadata
* Any plans to improve this use case in the spec?

**Comment by stain** [comment]
As discussed in Slack:

See also https://bioschemas.org/types/LabProtocol/0.5-DRAFT and https://bioschemas.org/types/LabProcess/0.1-DRAFT
and how SEEK does this in https://fairdomhub.org/sops  and their corresponding assays (e.g. an "execution" of a SOP).  Some presentation in a recent workshop by  Nick Juty in https://bit.ly/WFH_SOPs

We also use `HowToStep` and `HowTo` in https://www.researchobject.org/workflow-run-crate/profiles/provenance_run_crate/ -- these are generally meant for manual steps so should be well suited as well for SOPs. 

Would you have suggestions for a new section on using https://schema.org/HowTo in this way and connecting it to `CreateAction` for a physical sample/specimen?
